pkg/controller/kogitoapp/kogitoapp_controller.go

```diff
@@ -83,32 +112,14 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
-	err = addWatchObjects(c,
-		&handler.EnqueueRequestForOwner{IsController: true, OwnerType: &v1alpha1.KogitoApp{}},
-		&oappsv1.DeploymentConfig{},
-		&corev1.Service{},
-		&routev1.Route{},
-		&obuildv1.BuildConfig{},
-		&oimagev1.ImageStream{},
-		&corev1.ConfigMap{})
-	if err != nil {
+	controllerWatcher = framework.NewControllerWatcher(r.(*ReconcileKogitoApp).client, mgr, c, &v1alpha1.KogitoApp{})
+	if controllerWatcher.Watch(watchedObjects...); err != nil {
```

Comment:
The err is not assigned here, so you don't have to check it.

Reply:
Sorry, the Watch returns err...

Reply:
LOL, copy and paste.